AgileBoard.Api/src/AgileBoard.Api/Services/UserStoryService.cs

```diff
@@ -0,0 +1,103 @@
+﻿using AgileBoard.Api.Domain;
+using AgileBoard.Api.Logging;
+using AgileBoard.Api.Repositories;
+using Microsoft.AspNetCore.JsonPatch;
+
+namespace AgileBoard.Api.Services
+{
+    public class UserStoryService : IUserStoryService
+    {
+        private readonly IUserStoryRepository _userStoryRepository;
+        private readonly ILoggerAdapter<UserStoryService> _logger;
+
+        public UserStoryService(IUserStoryRepository userStoryRepository, ILoggerAdapter<UserStoryService> logger)
+        {
+            _userStoryRepository = userStoryRepository;
+            _logger = logger;
+        }
+
+        public async Task<IEnumerable<UserStory>> GetUserStoriesAsync()
+        {
+            _logger.LogInformation("Retrieving all user stories");
```

Comment:
First of all, you don't want to purify it like that :). Use decorator pattern if you want to and exception handling middleware so you don't catch exception everywhere